version.txt

```diff
@@ -1 +1 @@
-1.0.1
+1.0.2
```

Comment:
I believe this PR actually warrants at least a minor version bump due to the changes. Major version bump is a toss up based on how you view pulling `develop` vs. a specific tag by default with regard to backwards-compatibility.